tests/php/3rd-party/test_class-domain-mapping.php

```diff
@@ -1,30 +1,78 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase, WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Tests for the 3rd-party domain mapping plugin integration.
+ *
+ * @package Jetpack
+ * @phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
+ */
 
-use Automattic\Jetpack\Constants;
+namespace Automattic\Jetpack\Third_Party;
 
-require_once JETPACK__PLUGIN_DIR . '3rd-party/domain-mapping.php';
+use Automattic\Jetpack\Constants;
 
-// Extend with a public constructor so we can test
-class MockDomainMapping extends Jetpack_3rd_Party_Domain_Mapping  {
+require_once JETPACK__PLUGIN_DIR . '3rd-party/class-domain-mapping.php';
+
+/**
+ * Class MockDomainMapping
+ *
+ * Extend with a public constructor so we can test.
+ *
+ * @package Jetpack
+ */
+class MockDomainMapping extends Domain_Mapping {
+	/**
+	 * MockDomainMapping constructor.
+	 */
 	public function __construct() {
 	}
 }
 
-class WP_Test_Domain_Mapping extends WP_UnitTestCase {
-	function tearDown() {
+/**
+ * Class WP_Test_Domain_Mapping
+ *
+ * @package Jetpack
+ */
+class WP_Test_Domain_Mapping extends \WP_UnitTestCase {
+	/**
+	 * String holding the name to pass to the MockBuilder.
+	 *
+	 * @var string
+	 */
+	private static $mock_builder_name;
+
+	/**
+	 * Test Tear down.
+	 */
+	public function tearDown() {
 		Constants::clear_constants();
 		foreach ( $this->get_jetpack_sync_filters() as $filter ) {
 			remove_all_filters( $filter );
 		}
+
+		parent::tearDown();
+	}
+
+	/**
+	 * Test Setup.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		self::$mock_builder_name = __NAMESPACE__ . '\\MockDomainMapping';
```

Comment:
```suggestion
		self::$mock_builder_name = MockDomainMapping::class;
```
Or maybe even better, lose the variable and just use `MockDomainMapping::class` directly where it's needed.

Reply:
Ah, yes. That's much cleaner as suggested and using directly also makes sense to me. Removes the need for the var and the `setUp` method.

Implemented in d116a942b